assets/artifact-hub-pkg/package.go

```diff
@@ -108,9 +108,9 @@ func slugify(name string) string {
 	return s
 }
 func fetchCatalogPatterns() ([]byte, error) {
-	resp, err := http.Get(fmt.Sprintf("%s/api/catalog/content/pattern", mesheryCloudBaseURL))
+	resp, err := http.Get(fmt.Sprintf("%s/api/catalog/content/pattern?page=0&pageSize=0", mesheryCloudBaseURL))
```

Comment:
Using page=0 causes internal server error. And to begin with if both are zero then nothing would be returned.

Reply:
That's right. I was thinking the same thing at first, but then when I was testing locally, it did return designs, so I thought it shouldn't matter and that 0 might be the default value(Wrong assumption on my part I guess). There was a PR merged recently, [#2854](https://github.com/layer5io/meshery-cloud/pull/2854?notification_referrer_id=NT_kwDOAziWILQxMjk5MDY2MTYyMTo1NDA0MDA5Ng#event-14860254179). In this PR, if the pagesize is 0, then ideally, all the patterns in the catalog should be returned.
Here's my response from testing it using postman(with parameters  0 & 0)
[responsePostman.txt](https://github.com/user-attachments/files/17545087/responsePostman.txt)

Also, I only had 4 designs locally so that might be the case its working here but not on production.

Reply:
@TraXIcoN feedback incorporated?

Reply:
@TraXIcoN feedback incorporated?

Reply:
@Jougan-0 will you please finish this off on behalf of @TraXIcoN?

Reply:
The issue had already been fixed. Closing this PR.